core/templates/dev/head/profile/profile.html

```diff
@@ -33,8 +33,15 @@
   <div class="oppia-profile-container" ng-controller="Profile">
     <md-card class="oppia-profile-user-card">
 
+    <div ng-if=" '{{username}}' == '{{PROFILE_USERNAME}}' ">
```

Comment:
The {{...}} is Jinja syntax, and the ng-if is Angular. I don't think we should mix the two.

Instead, can we do everything in Angular? In Profile.js, declare a $scope.profileIsOfCurrentUser and set it to false at the outset. When the data is retrieved from the backend, compare GLOBALS.PROFILE_USERNAME with data.profile_username and if it matches, set that variable to true. Then, in the HTML, do ng-if="profileIsOfCurrentUser".